### ISSUE TITLE
Should authors be able to distinguish between open and closed captions?

### ISSUE THREAD

**Issue by jasonjgw** [w3c/a11y-discov-vocab#26]
Should there be an optional extension for distinguishing between open and closed captions?
Examples: "captions/open", "captions/closed".
Does this matter in describing media resources?

**Comment by madeleinerothberg** [comment]
Yes, open vs closed captions should be distinguished if known. Add this to our list of "how to note variations/extensions/refinements".

**Comment by mattgarrish** [comment]
Should we just add `openCaptions` and `closedCaptions` as values?

That would be consistent with our current spec guidance to create more specialized terms when appropriate.
